scripts/test.sh

```diff
@@ -54,14 +50,13 @@ function run_flake8 {
     return 0
   fi
 
-  flake8 scripts
+  for script in $1; do
+    flake8 $script
+  done
```

Comment:
I'm somewhat confused here. This implies an argument is now passed to this function. Where does it come from?